app/views/static_pages/home.html.erb

```diff
@@ -9,8 +9,8 @@
     </div>
   </div>
 </div>
-<h2 class="my-5"><%= t "rooms.all_rooms" %></h2>
+<h2 class="my-5"><%= params[:key] ? t("other.search_results", key: params[:key]) : t("rooms.all_rooms") %></h2>
 <div class="row row-cols-1 row-cols-md-3 g-4 mt-4">
-  <%= render @rooms %> 
+  <%= @rooms.present?  ? render(@rooms)  : t("other.search_not_matching") %>
```

Comment:
`@rooms.present?` -> có đúng ko em?
test đi

em tìm hiểu về method `any?` nhé

ngoài ra 
ko cần toán tử 3 ngôi đâu
```suggestion
  <%= render(@rooms) || t("other.search_not_matching") %>
```

https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections